fastly/resource_fastly_tls_subscription.go

```diff
@@ -330,3 +331,17 @@ func resourceFastlyTLSSubscriptionDelete(_ context.Context, d *schema.ResourceDa
 func resourceFastlyTLSSubscriptionStateNotIssued(_ context.Context, d *schema.ResourceDiff, _ interface{}) bool {
 	return d.Get("state").(string) != "issued"
 }
+
+func resourceFastlyTLSSubscriptionSetNewComputed(_ context.Context, d *schema.ResourceDiff, _ interface{}) error {
+	// NOTE: This is a workaround for a bug in Terraform core
```

Comment:
Can we link to an issue? I presume there's an open issue on the Hashicorp Terraform GitHub repo?

Reply:
I added the link in the description above. It's still my _assumption_ so I'm trying to clarify that part.